tests/test_autonomy/test_cli/test_fetch.py

```diff
@@ -24,13 +24,16 @@
 from pathlib import Path
 from unittest import mock
 
+import pytest
+
 from tests.conftest import ROOT_DIR
 from tests.test_autonomy.test_cli.base import BaseCliTest
 
 
 IPFS_REGISTRY = "/dns/registry.autonolas.tech/tcp/443/https"
 
 
+@pytest.mark.skip  # TOFIX
```

Comment:
I skipped this test because this depends on the service component being published every time it gets updates which is not ideal, so I'll update this test to work with a dummy service which never changes.

Reply:
Open issues always please